test/e2e/action_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build integration
```

Comment:
In order to run integration tests, this change is needed: https://github.com/capactio/hub-manifests/pull/56